### PR TITLE
keyword bug fixed

### PR DESCRIPTION
**Pull request by tjmgd**
In the entity outliner search box, the matching results are not expanded and displayed (prefab mode) 

Signed-off-by: T.J. McGrath-Daly <tj.mcgrath.daly@huawei.com>

### CODE REVIEW

**Comment by hultonha** [comment]
Hi @tjmgd, would you please be able to add some more detail/context to the description for what this change is fixing (what was the initial issue, what do things look like now etc...). The additional context makes it a lot easier to understand what the change is doing. Also if this is related to an earlier PR then linking it here can also be helpful if this is a cherry-pick for example. Thanks!

**Comment by tjmgd** [comment]
Updated description. This is one of a pair of PRs with a similar theme. The other is here:
https://github.com/o3de/o3de/pull/6240

**Comment by hultonha** [comment]
Awesome thanks @tjmgd! 👍 Makes sense now cheers 🙂 Great I'll check out the other now too

**Review by hultonha — APPROVED** [review]
This makes sense to me but it would be great if @cgalvan and/or @AMZN-daimini could take a quick look in case there's anything else that might be impacted by this.

Also I wonder if @jckand-amzn might know if there's a way to test this using the Python automation work. This PR has something similar - https://github.com/o3de/o3de/pull/6036/files but for the Asset Browser.

**Review by AMZN-daimini — APPROVED** [review]
Makes sense to me, don't think it impacts other features.

**Comment by tjmgd** [comment]
I am a little confused as to why the build failed here. Any guidance would be appreciated.

**Comment by hultonha** [comment]
It looks like a comment from @cgalvan is blocking on this change for the moment - https://github.com/o3de/o3de/pull/6240#discussion_r765018879 so we might need to wait for that to be resolved before merging...

Looking at the failure it seems like it was a Jenkins connection issue most likely, there could have been a brief outage of some reason... A fresh build should resolve it but we'll wait for the comment Chris has to be resolved.

**Comment by AMZN-daimini** [comment]
> I am a little confused as to why the build failed here. Any guidance would be appreciated.

I have started a new AR run

**Pull request merged**
